addons/mail/static/src/models/time/time.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'Time',
+    identifyingFields: ['messaging'],
+    lifecycleHooks: {
+        _created() {
+            // Update the minute field every minutes
+            this._everyMinuteIntervalId = setInterval(
+                () => this.update({ currentDateEveryMinute: new Date() }),
+                60 * 1000
+            );
+        },
+        _willDelete() {
+            clearInterval(this._everyMinuteIntervalId);
+        }
+    },
+    fields: {
+        messaging: one2one('Messaging', {
+            inverse: 'time',
+            required: true,
+            readonly: true,
+        }),
+        currentDateEveryMinute: attr({
+            default: new Date(),
+        }),
```

Comment:
Missing documentation, something like
_Contains a Date object that is set to the current time, and is (re-)computed every minute._